src/cloud/components/DocPage/DocContextMenuActions.tsx

```diff
@@ -70,6 +71,7 @@ export function DocContextMenuActions({
   currentUserIsCoreMember,
   editorRef,
   restoreRevision,
+  isCanvas,
```

Comment:
I think rather the opposite in this case there in terms of liability, the inCanvas flag is controlled directly from the page itself so we know what we are getting. 

if we base it on `doc.rootBlock` it exposes to our map updates, sse updates, and possible issues we did not foresee as we did not test enough in a team environment. We never had the best update flow out there so that's why I went this route for now.